pkgs/desktops/plasma-5/plasmoids/ddcci-plasmoid/ddcci-plasmoid-backend/default.nix

```
@@ -0,0 +1,43 @@
+{ lib
+, fetchPypi
+, python3Packages
+, ddcutil
+}:
+python3Packages.buildPythonPackage rec {
+  pname = "ddcci_plasmoid_backend";
+  version = "0.1.10";
+
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit pname version;
```

```suggestion
  pname = "ddcci-plasmoid-backend";
  version = "0.1.10";

  format = "pyproject";

  src = fetchPypi {
    pname = "ddcci_plasmoid_backend";
    inherit version;
```
see python language docs in nixpkgs